### ISSUE TITLE
Amazon ClearUrls rule not sufficiently shortening url

### ISSUE THREAD

**Issue by shaun-mathew** [ClearURLs/Rules#51]
Trying the following url:

https://www.amazon.ca/ECOVACS-X1-Self-Empty-Auto-Clean-Navigation/dp/B09NPKWDRX/ref=sr_1_1_sspa?crid=3D1K4MEK1XDXA&keywords=x1+omni+vacuum&qid=1659903328&sprefix=omni+vac%2Caps%2C98&sr=8-1-spons&psc=1&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyNkQyWERQWUtGRDNZJmVuY3J5cHRlZElkPUEwMzIwNDg3MUxCNUFJSU1SMUlVMCZlbmNyeXB0ZWRBZElkPUExMDQyNTk1M0QxWFhJVFJLMzdRUCZ3aWRnZXROYW1lPXNwX2F0ZiZhY3Rpb249Y2xpY2tSZWRpcmVjdCZkb05vdExvZ0NsaWNrPXRydWU=

only shortens to this:

https://www.amazon.ca/ECOVACS-X1-Self-Empty-Auto-Clean-Navigation/dp/B09NPKWDRX?psc=1&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyNkQyWERQWUtGRDNZJmVuY3J5cHRlZElkPUEwMzIwNDg3MUxCNUFJSU1SMUlVMCZlbmNyeXB0ZWRBZElkPUExMDQyNTk1M0QxWFhJVFJLMzdRUCZ3aWRnZXROYW1lPXNwX2F0ZiZhY3Rpb249Y2xpY2tSZWRpcmVjdCZkb05vdExvZ0NsaWNrPXRydWU%3D

This does not sufficiently shorten the url. It can be further shortened to

https://www.amazon.ca/ECOVACS-X1-Self-Empty-Auto-Clean-Navigation/dp/B09NPKWDRX

**Comment by Splarkszter** [comment]
Yeah, clearurls seems to only functions well with "Amazon.com" but not it's derivatives. Quite a bummer, this should be changed/improved.
